app/views/galleries/_expandable.haml

```diff
@@ -12,16 +12,16 @@
   %td.width-70.gallery-buttons{class: klass}
     - if gallery.user_id == current_user.try(:id)
       = link_to add_gallery_path(gallery), title: "Add Icons", class: 'gallery-add' do
-        = image_tag "icons/add.png"
+        = image_tag "icons/add.png", alt: 'Add Icons'
       = link_to edit_gallery_path(gallery), title: "Edit Gallery", class: 'gallery-edit' do
-        = image_tag "icons/pencil.png"
+        = image_tag "icons/pencil.png", alt: 'Edit'
       = link_to gallery_path(gallery), title: "Delete Gallery", method: :delete, data: { confirm: "Are you sure you want to delete #{gallery.name}?" }, class: 'gallery-delete' do
-        = image_tag "icons/cross.png"
+        = image_tag "icons/cross.png", alt: 'Delete'
   %td.centered.width-70{class: klass}
     .gallery-box.vmid{id: "minmax-#{gallery.id}", style: 'display: inline-block;', data: {id: gallery.id}}
       .view-button{style: 'padding: 5px 7px;'}
-        = image_tag "icons/bullet_arrow_down.png", class: 'vmid down-arrow'
-        = image_tag "icons/bullet_arrow_up.png", class: 'vmid hidden up-arrow'
+        = image_tag "icons/bullet_arrow_down.png", class: 'vmid down-arrow', alt: 'Display'
```

Comment:
I'd probably go with "Show", since Show/Hide tend to be a common pair (and Display can be a noun too)

Reply:
Same for the other Display/Hide pairs